moveit.rosinstall

```diff
@@ -24,3 +24,7 @@
     local-name: srdfdom
     uri: https://github.com/ros-planning/srdfdom.git
     version: kinetic-devel
+- git:
+    local-name: industrial_moveit
+    uri: https://github.com/raghavendersahdev/industrial_moveit.git
```

Comment:
this needs to be switched to the new stomp repo that is suppose to be created. fyi @gavanderhoorn